am-ui/webcomponents/create-ad-webcomponent/package.json

```diff
@@ -11,30 +11,66 @@
     "preview": "vite preview",
     "openapi-gen": "openapi-generator-cli generate -g javascript -i ../../../api.yaml -o ./src/api",
     "package": "node package-script.js",
-    "lint": "eslint --ext .js,.vue --ignore-path .gitignore --fix src"
+    "test:unit": "vue-cli-service test:unit",
+    "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@babel/core": "^7.17.8",
-    "@babel/eslint-parser": "^7.17.0",
-    "@babel/preset-env": "^7.16.11",
     "@openapitools/openapi-generator-cli": "^2.4.26",
     "@vitejs/plugin-vue-jsx": "^1.3.8",
     "concat": "^1.0.3",
-    "eslint-plugin-jsonc": "^2.2.1",
-    "eslint-plugin-node-dependencies": "^0.8.0",
+    "core-js": "^3.8.3",
+    "ts-jest": "^27.1.4",
+    "vue": "^3.2.13",
     "superagent": "^7.1.1",
-    "vue": "^3.2.25",
     "vue-i18n": "^9.2.0-beta.34"
   },
   "devDependencies": {
-    "@babel/preset-react": "^7.16.7",
     "@intlify/vite-plugin-vue-i18n": "^2.4.0",
     "@vitejs/plugin-vue": "^2.2.0",
-    "eslint": "^8.12.0",
-    "eslint-config-google": "^0.14.0",
-    "eslint-plugin-eslint-plugin": "^4.1.0",
-    "eslint-plugin-vue": "^8.5.0",
-    "vite": "^2.8.0",
-    "vue": "^3.2.25"
-  }
+    "@vitejs/plugin-vue-jsx": "^1.3.9",
+    "autoprefixer": "^10.4.4",
+    "@babel/core": "^7.12.16",
+    "@babel/eslint-parser": "^7.12.16",
+    "@vue/cli-plugin-babel": "~5.0.0",
+    "@vue/cli-plugin-eslint": "~5.0.0",
+    "@vue/cli-plugin-unit-jest": "~5.0.0",
+    "@vue/cli-service": "~5.0.0",
+    "@vue/test-utils": "^2.0.0-0",
+    "@vue/vue3-jest": "^27.0.0-alpha.1",
+    "babel-jest": "^27.0.6",
+    "eslint": "^7.32.0",
+    "eslint-plugin-vue": "^8.0.3",
+    "jest": "^27.0.5"
+  },
+  "eslintConfig": {
```

Comment:
Why is there another eslintConfig declaration here, shouldn't we use `.eslintrc.js`?

Reply:
Also true. I just had so many problems configuring the tests that this I didn't check either once was working. Checked without and all good so I removed it